doc/rtd/explanation/failure_states.rst

```diff
@@ -0,0 +1,74 @@
+.. _failure_states:
```

Comment:
Could be, but while related they are really different concepts. One is about the states of cloud-init, the other is related to extracting information if cloud-init is in one of those states. I'd rather keep them separate for now, but I'll add a link to the page since it isn't defined here - no need to re-define.